keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -210,21 +189,21 @@ metadata:
   finalizers:
   - cosi.io/finalizer [2]
 spec:
-  requestProtocol: [3]
+  protocol: [3]
   bucketPrefix: [4]
   bucketClassName: [5]
-  secretName: [6]
+  accessSecretName: [6]
```

Comment:
would `userSecretName` be more consistent here? Thinking it matches the BucketClass `cosi.io/user-secret-name` keys.